addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/KodiBindingConstants.java

```diff
@@ -38,6 +38,8 @@
     public static final String CHANNEL_STOP = "stop";
     public static final String CHANNEL_CONTROL = "control";
     public static final String CHANNEL_PLAYURI = "playuri";
+    public static final String CHANNEL_PVROPENTV = "pvropentv";
+    public static final String CHANNEL_PVROPENRADIO = "pvropenradio";
```

Comment:
Because it makes it readable
```java
public static final String CHANNEL_PV_ROP_EN_TV = "pv-rop-en-tv";
```

Or should it be 
```java
public static final String CHANNEL_PVR_OPEN_TV = "pvr-open-tv";
```